parlai/tasks/twitter/build.py

```diff
@@ -8,8 +8,8 @@
 try:
     from emoji.unicode_codes import UNICODE_EMOJI
     import unidecode
-except ModuleNotFound:
```

Comment:
this is not related to tensorboard, but also a bug

Reply:
nice